apps/web/src/views/AddLiquidityV3/components/V3SubmitButton.tsx

```diff
@@ -80,7 +80,7 @@ export function V3SubmitButton({
           isValid && (
             <RowBetween style={{ gap: '8px' }}>
               {showApprovalA && (
-                <Button onClick={() => approveACallback()} disabled={approvalA === ApprovalState.PENDING} width="100%">
+                <Button onClick={approveACallback} disabled={approvalA === ApprovalState.PENDING} width="100%">
```

Comment:
We need to maintain `() => approveACallback()` right?
This fix by Jojo yesterday.

https://github.com/pancakeswap/pancake-frontend/pull/7816